org.eclipse.sumo.metainfo.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should be contributed upstream.

Reply:
I submit a patch to them, just some contributor agreement to do.